server/websocket.go

```diff
@@ -1230,18 +1230,23 @@ func (c *client) wsCollapsePtoNB() (net.Buffers, int64) {
 			for i := 0; i < len(nb); i++ {
 				b := nb[i]
 				if total+len(b) <= mfs {
-					bufs = append(bufs, b)
+					bufs = append(bufs, copyIfMasking(mask, b))
```

Comment:
The copy here still needed?

Reply:
I guess we may not need since this is a buffer for a particular client and it is unique to that client (has been copied during queueOutbound), and the buffer is discarded after being sent. Let me revisit that...

Reply:
I have removed that and updated PR (with force push)...

Reply:
(also rebased from main branch prior to force push)